Source/WebCore/Modules/webauthn/AttestationConveyancePreference.h

```diff
@@ -28,6 +28,7 @@
 #if ENABLE(WEB_AUTHN)
 
 #include <wtf/Forward.h>
+#include <wtf/text/WTFString.h>
```

Comment:
You don't need this include. Forward already forward-declares; instead you should add WTFString to any file that compiles with this header (if it doesn't already).